Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -53,15 +54,16 @@ deriving BEq
 inductive StyleError where
   /-- Missing or malformed copyright header.
   Unlike in the python script, we may provide some context on the actual error. -/
-  | copyright (context : Option String)
+  | Copyright (context : Option String)
   /-- Malformed authors line in the copyright header -/
-  | authors
+  | Authors
   /-- The bare string "Adaptation note" (or variants thereof): instead, the
   #adaptation_note command should be used. -/
-  | adaptationNote
+  | AdaptationNote
   /-- Lint against "too broad" imports, such as `Mathlib.Tactic` or any module in `Lake`
   (unless carefully measured) -/
-  | broadImport (module : BroadImports)
+  | BroadImport (module : BroadImports)
+  | DuplicateImport (importStatement: String) (alreadyImportedLine: ℕ)
```

Comment:
I don't think this is following the naming convention, is it? My intuition says `StyleError.copyright` should be lower case

Reply:
Indeed, I had referred to the capitalisation of the *error messages*, not the constructors. Thanks for correcting this.